src/clients/nfs_server.rb

```diff
@@ -115,14 +116,16 @@ def main
             )
           },
           "enablev4"   => {
-            "type" => "string",
-            "help" => _(
-              "Yes/No option for enabling/disabling support for NFSv4."
+            "type"     => "enum",
+            "typespec" => ["yes", "no"],
+            "help"     => _(
+              "'yes'/'no option for enabling/disabling support for NFSv4."
```

Comment:
Missing quote

```suggestion
              "'yes'/'no' option for enabling/disabling support for NFSv4."
```

BTW do we support *translated* input, e.g. does it handle the Czech `ano`, `ne` as the input or only the English `yes`, `no` are expected?

If translations are not supported we should rather do something like

```ruby
_("%{yes}'/'%{no}' option for enabling/disabling support for NFSv4.") % { yes: "yes", no: "no" }
```

I'm afraid that the translators will also translate that "yes/no"...

Reply:
yes, that makes sense. We do not support translated input and to be honest it is hard to believe that someone really use that translated output in CLI which we target mainly for scripting.